packages/components/src/FocusManager/FocusManager.component.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * This component handle some specifics case like when we click in the datalist
```

Comment:
Typo 
-`handle` -->`handles`
- `specifics case` --> `specific cases`

Your description is good as example. Can you add a little generic description before, explaining that this allows to  capture the blur that occurs outside of complex components ?